code/datums/datacore.dm

```diff
@@ -65,13 +65,9 @@
 			crimes |= crime
 			return
 

```

Comment:
I am going to presume this will cause a fair amount of bugs. We should test merge to check it.
If it dosent okay its probably old issye reasons why.